resources/about.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Octopeer</title>
+
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link rel="stylesheet" href="../libs/materialize/css/materialize.min.css">
+    <link rel="stylesheet" href="main.css">
+
+    <script src="../libs/jquery-2.2.3.js"></script>
+    <script src="../libs/materialize/js/materialize.min.js"></script>
+    <script src="about.js"></script>
+
+</head>
+<body>
+
+<nav>
+    <div class="nav-wrapper lime">
+        <div class="brand-logo center" style="height: auto; width: auto;">
+            <a>Octopeer</a>
+        </div>
+
+        <ul class="right hide-on-small-only">
+            <li><a href="../analytics/index.html">Analytics</a></li>
+            <li><a href="options.html">Options</a></li>
+            <li class="active"><a href="#">About</a></li>
+        </ul>
+
+        <ul id="slide-out" class="side-nav">
+            <li><a href="../analytics/index.html">Analytics</a></li>
+            <li><a href="options.html">Options</a></li>
+            <li class="active"><a href="#">About</a></li>
+        </ul>
+        <a href="#" data-activates="slide-out" class="right button-collapse hide-on-med-and-up"><i class="material-icons">menu</i></a>
+
+    </div>
+</nav>
+
+<div class="container">
+    <p class="flow-text" style="text-align: center;">
+        This page tells you all about Octopeer.
+    </p>
+
+    <div class="row">
+
+        <div class="col s12 m6">
+            <p class="flow-text">What is Octopeer?</p>
+            <p>
+                Octopeer is a chrome extension that will help you analyze your personal code reviewing behaviour.
+                While reviewing code in pull requests on both GitHub and BitBucket this extension will collect data.
```

Comment:
Well.... it is XD In that case maybe reference to the BitBucket extension instead of giving the impression that this extension does both?